manifests/clusterresourceoverride-operator.package.yaml

```diff
@@ -4,3 +4,5 @@ channels:
     currentCSV: clusterresourceoverride-operator.v4.4.0
   - name: "4.5"
     currentCSV: clusterresourceoverride-operator.v4.5.0
+  - name: "4.6"
```

Comment:
@rphillips try removing both 4.4 and 4.5 channels from the package, this should set 4.6 as the default channel. 

If there is only one channel in a package then it is considered as the default channel - https://github.com/operator-framework/operator-registry/blob/bf0c8f1590d21e54e97c1446b09fae72ade51280/pkg/registry/types.go#L82